monitoring/tests/system/test_vpcsc_v3.py

```diff
@@ -21,630 +21,397 @@
 
 import os
 import pytest
-import logging
+
 from google.api_core import exceptions
 from google.cloud import monitoring_v3
 from google.cloud.monitoring_v3 import enums
-
-PROJECT_INSIDE = os.environ.get("PROJECT_ID", None)
-PROJECT_OUTSIDE = os.environ.get(
-    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT", None
-)
-IS_INSIDE_VPCSC = os.environ.get("GOOGLE_CLOUD_TESTS_IN_VPCSC", "false")
+from test_utils.vpcsc_config import vpcsc_config
 
 
+@vpcsc_config.skip_unless_inside_vpcsc
 class TestVPCServiceControlV3(object):
     @staticmethod
     def _is_rejected(call):
-        logger = logging.getLogger(__name__)
         try:
-            responses = call()
-
-            # If we reach this line, then call() did not raise. The return
-            # result must be either a google.api_core.page_iterator.Iterator
-            # instance, or None.
-            list(responses)
+            call()
         except exceptions.PermissionDenied as e:
-            logger.debug(e)
             return e.message == "Request is prohibited by organization's policy"
-        except Exception as e:
```

Comment:
These tests are written to just check that requests are blocked/allowed in the VPC environment. A lot of them are invalid requests and will fail for other reasons, which is why this extra block is here.

Reply:
Please see the updated module in e403c1b :  there, I make actual assertions of the expected errors.